lib/src/signed_video_h26x_common.c

```diff
@@ -864,10 +864,47 @@ compute_partial_gop_hash(signed_video_t *self)
     // The list index is zero, which is means list is empty and there is nothing to compute.
     return SV_OK;
   }
-
+#ifdef SIGNED_VIDEO_DEBUG
+  printf("size :%d \n compute_partial_gop_hash\n", gop_info->list_idx);
+  for (int i = 0; i < gop_info->list_idx; i++) {
+    printf("%02x", gop_info->hash_list[i]);
+  }
+  printf("\n");
+#endif
   return openssl_hash_data(self->crypto_handle, gop_info->hash_list, gop_info->list_idx, hash);
 }
 
+/* compute_partial_gop_hash()
+ * Takes all the NALU hashes from |nalu_hash_list| and hash it.
+ */
+svrc_t
+compute_partial_gop_hash_for_auth(signed_video_t *self)
```

Comment:
The operation is (and should be) identical to `compute_partial_gop_hash(...)`. To make sure these functions do not deviate from each other I would like to suggest to combine them into one function as `compute_partial_gop_hash(signed_video_t *self, const uint8_t *hash_list, size_t hash_list_size)`